### ISSUE TITLE
Issue plotting ROI with navis.plot3d()

### ISSUE THREAD

**Issue by ASiliciano** [schlegelp/navis#12]
Following the neuprint tutorial:

```
import navis
# Make a 3D plot
fig = navis.plot3d([mbon_skeletons[0], mb])
```

I get this error :
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-9-7726b8aa70a7> in <module>
      2 
      3 # Make a 3D plot
----> 4 fig = navis.plot3d([mbon_skeletons[0], mb])

/opt/anaconda3/lib/python3.7/site-packages/navis/plotting/ddd.py in plot3d(x, **kwargs)
    219         return plot3d_vispy(x, **kwargs)
    220     else:
--> 221         return plot3d_plotly(x, **kwargs)
    222 
    223 

/opt/anaconda3/lib/python3.7/site-packages/navis/plotting/ddd.py in plot3d_plotly(x, **kwargs)
    311         data += neuron2plotly(neurons, **kwargs)
    312     if volumes:
--> 313         data += volume2plotly(volumes, **kwargs)
    314     if points:
    315         scatter_kws = kwargs.pop('scatter_kws', {})

/opt/anaconda3/lib/python3.7/site-packages/navis/plotting/plotly/graph_objs.py in volume2plotly(x, **kwargs)
    379                                       volumes=x,
    380                                       alpha=kwargs.get('alpha', None),
--> 381                                       color_range=255)
    382 
    383     trace_data = []

ValueError: not enough values to unpack (expected 3, got 2)
```
The issue may be with the mb volume ROI as the following works well:

`fig = navis.plot3d([mbon_skeletons[0]])`

**Comment by schlegelp** [comment]
Hi & Thanks for the bug report - that one slipped past me. Fixed with 33df7e2168431f9ccf7e28a40ada05c2faf7bac0.

I also pushed a new version (`0.3.3`) to PyPI and if you update navis (`pip3 install navis -U`), you should be able to run the example.

**Comment by ASiliciano** [comment]
Thanks!  That took care of it
